nixos/modules/services/monitoring/perses.nix

```diff
@@ -0,0 +1,128 @@
+{
+  pkgs,
+  lib,
+  config,
+  utils,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkOption
+    mkEnableOption
+    mkPackageOption
+    mkIf
+    types
+    ;
+
+  cfg = config.services.perses;
+
+  settingsFormat = pkgs.formats.yaml { };
+
+in
+{
+  options.services.perses = {
+    enable = mkEnableOption "perses";
+
+    package = mkPackageOption pkgs "perses" { };
+
+    port = mkOption {
+      type = types.port;
+      default = 8080;
+      description = ''
+        Perses Web interface port.
+      '';
+    };
+
+    listenAddress = mkOption {
+      type = types.str;
+      default = "";
```

Comment:
If left unchanged, does this listen on `0.0.0.0`? Would it be better to set that as the explicit default (or possibly `127.0.0.1`)?

Reply:
see https://github.com/NixOS/nixpkgs/pull/411771#discussion_r2384123606

Reply:
Maybe a comment could be added to the description, explaining the default?

Reply:
Comment added.